From ltsi-dev-bounces@lists.linuxfoundation.org Mon Jun 18 02:55:43 2012
From: Tetsuyuki Kobayashi <koba@kmckk.co.jp>
Date: Mon, 18 Jun 2012 18:55:45 +0900
Subject: LTSI: bugfix: add CONFIG_I2C to kzm9g_defconfig
To: ltsi-dev@lists.linuxfoundation.org
Message-ID: <4FDEFB21.1090608@kmckk.co.jp>



Build fails for kzm9g because CONFIG_I2C=y is missing in kzm9g_defconfig.

$ make kzm9g_defconfig
$ make

  ...

  CC      arch/arm/mach-shmobile/board-kzm9g.o
arch/arm/mach-shmobile/board-kzm9g.c: In function 'as3711_enable_lcdc_backlight':
arch/arm/mach-shmobile/board-kzm9g.c:314:9: error: implicit declaration of function 'i2c_get_adapter' [-Werror=implicit-function-declaration]
arch/arm/mach-shmobile/board-kzm9g.c:314:26: warning: initialization makes pointer from integer without a cast [enabled by default]
arch/arm/mach-shmobile/board-kzm9g.c:349:3: error: implicit declaration of function 'i2c_transfer' [-Werror=implicit-function-declaration]
cc1: some warnings being treated as errors
make[1]: *** [arch/arm/mach-shmobile/board-kzm9g.o] Error 1
make: *** [arch/arm/mach-shmobile] Error 2

This patch solves this problem.

Signed-off-by: Tetsuyuki Kobayashi <koba@kmckk.co.jp>
---
 arch/arm/configs/kzm9g_defconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/kzm9g_defconfig b/arch/arm/configs/kzm9g_defconfig
index e3ebc20..ff7b698 100644
--- a/arch/arm/configs/kzm9g_defconfig
+++ b/arch/arm/configs/kzm9g_defconfig
@@ -78,6 +78,7 @@ CONFIG_SERIAL_SH_SCI=y
 CONFIG_SERIAL_SH_SCI_NR_UARTS=9
 CONFIG_SERIAL_SH_SCI_CONSOLE=y
 # CONFIG_HW_RANDOM is not set
+CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=y
 CONFIG_I2C_SH_MOBILE=y
 CONFIG_GPIO_PCF857X=y
-- 
1.7.9.5
_______________________________________________
LTSI-dev mailing list
LTSI-dev@lists.linuxfoundation.org
https://lists.linuxfoundation.org/mailman/listinfo/ltsi-dev

